WordPress/Classes/Utility/ZendeskUtils.swift

```diff
@@ -31,19 +33,72 @@ import ZendeskSDK
         ZendeskUtils.sharedInstance.enableZendesk(true)
     }
 
-    static func isZendeskEnabled() -> Bool {
-        return UserDefaults.standard.bool(forKey: UserDefaultsKeys.zendeskEnabled)
+    static func createIdentity(with accountSettings: AccountSettings) {
+        let zendeskIdentity = ZDKAnonymousIdentity()
+
+        var userName = accountSettings.username
+        if accountSettings.firstName.count > 0 || accountSettings.lastName.count > 0 {
+            userName = accountSettings.firstName + " " + accountSettings.lastName
```

Comment:
It looks like a user might choose to specify only a first name, or only a last name.  In this case the concatenation with the space might look a bit weird.  Maybe we could trim any leading/trailing white space as a bit of polish?